views/confirm-login.hbs

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width">
+  <title>Confirm GitHub Sign in</title>
+  <link rel="stylesheet" href="/static/css/style.css">
+  <link href="https://fonts.googleapis.com/css?family=Lato:400,700,900" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css?family=Roboto:300,400,700" rel="stylesheet">
+  <link rel="shortcut icon" href="/static/img/favicon.ico" />
+  <!-- Global site tag (gtag.js) - Google Analytics -->
+  <script async src="https://www.googletagmanager.com/gtag/js?id=UA-3769691-98"></script>
+  <script>
+    window.dataLayer = window.dataLayer || [];
+    function gtag() { dataLayer.push(arguments); }
+    gtag('js', new Date());
+
+    gtag('config', 'UA-3769691-98');
+  </script>
+</head>
+
+<body>
+
+  <div class="text-center">
+    <div class="container-lg p-responsive py-6 py-lg-10">
+      <img src="/static/img/githubplusslack.svg" class="mb-3 mb-sm-5 col-6 col-sm-12" style="max-width: 300px;">
+      <p>Finish connecting your GitHub account to your slack account on </p>
+      <p class="alt-lead text-gray col-md-8 mx-auto mb-3 mb-lg-4">
+        {{teamSlackDomain}}.slack.com
+      </p>
+      <p>You will be able to view rich previews for GitHub links, open and close issues, and much more from Slack</p>
```

Comment:
I'm thinking we might want to be explicit here on why this is required beyond just installing the app. It's not required to do this to view public GitHub links, right? So 👇 suggesting adding `private` and adding a bit about how it depends on what access you have on GitHub:

```suggestion
      <p>This will connect your accounts so you can use Slack to view rich previews for private GitHub links, open and close issues, and other features that depend on your access to GitHub!</p>
```

Reply:
So much better, thanks @tcbyrd ✨